README.md

```diff
@@ -254,23 +253,9 @@ var SignedXml = require("xml-crypto").SignedXml,
 Now define the extension point you want to implement. You can choose one or more.
 
 A key info provider is used to extract and construct the key and the KeyInfo xml section.
-Implement it if you want to create a signature with a KeyInfo section, or you want to read your key in a different way then the default file read option.
-
-```javascript
-function MyKeyInfo() {
-  this.getKeyInfo = function (key, prefix) {
-    prefix = prefix || "";
-    prefix = prefix ? prefix + ":" : prefix;
-    return "<" + prefix + "X509Data></" + prefix + "X509Data>";
-  };
-  this.getKey = function (keyInfo) {
-    //you can use the keyInfo parameter to extract the key in any way you want
-    return fs.readFileSync("key.pem");
-  };
-}
-```
+Implement it if you want to create a signature with a KeyInfo section, or you want to read your key in a different way then the built-in file-read or string methods. See the implementation in `string-key-info.js` for more information.
```

Comment:
The documentation on customizing the content of the KeyInfo element was removed.
There is only a reference to the `string-key-info.js` file which no longer exists.